tests/unit/api/bucketGetLogging.js

```diff
@@ -0,0 +1,298 @@
+            assert.strictEqual(err, null);
+
+            // The in-memory backend has account ID '123456789012' hardcoded
+            const accountId = '123456789012';
+            const getRequest = createGetLoggingRequest(bucketName, {
+                'x-amz-expected-bucket-owner': accountId,
+            });
+
+            bucketGetLogging(authInfo, getRequest, log, (err, xml) => {
+                assert.strictEqual(err, null);
+                assert(xml);
+                done();
+            });
+        });
+    });
+
+    it('should return error for mismatched expected bucket owner', done => {
+        const wrongAccountId = '999999999999';
+        const getRequest = createGetLoggingRequest(bucketName, {
+            'x-amz-expected-bucket-owner': wrongAccountId,
+        });
+
+        bucketGetLogging(authInfo, getRequest, log, err => {
+            assert(err);
+            assert.strictEqual(err.is.AccessDenied, true);
+            done();
+        });
+    });
+
+    it('should return error for unauthorized access', done => {
+        const loggingXML = createValidLoggingXML(targetBucket);
+        const putRequest = createLoggingRequest(bucketName, loggingXML);
+
+        // Set logging with owner
+        bucketPutLogging(authInfo, putRequest, log, err => {
+            assert.strictEqual(err, null);
+
+            // Try to get logging with different auth
+            const getRequest = createGetLoggingRequest(bucketName);
+            bucketGetLogging(otherAuthInfo, getRequest, log, err => {
+                assert(err);
+                assert.strictEqual(err.is.AccessDenied, true);
+                done();
+            });
+        });
+    });
+
+    it('should return error for non-existent bucket', done => {
+        const getRequest = createGetLoggingRequest('nonexistentbucket');
+
+        bucketGetLogging(authInfo, getRequest, log, err => {
+            assert(err);
+            assert.strictEqual(err.is.NoSuchBucket, true);
+            done();
+        });
+    });
+
+    it('should return well-formed XML response', done => {
+        const loggingXML = createValidLoggingXML(targetBucket);
+        const putRequest = createLoggingRequest(bucketName, loggingXML);
+
+        bucketPutLogging(authInfo, putRequest, log, err => {
+            assert.strictEqual(err, null);
+
+            const getRequest = createGetLoggingRequest(bucketName);
+            bucketGetLogging(authInfo, getRequest, log, (err, xml) => {
+                assert.strictEqual(err, null);
+                assert(xml);
+                // Try to parse the XML, if no error, it's well-formed
+                parseString(xml, (parseErr, result) => {
+                    assert.strictEqual(parseErr, null, `XML is not well-formed: ${parseErr}`);
+                    assert(result);
+                    done();
+                });
+            });
+        });
+    });
+
+    it('should handle multiple get requests consistently', done => {
```

Comment:
I don't know if this test is really useful, but no problem to keep it if you prefer.